roadrunner.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+# Namespace for all RoadRunner code
+module RoadRunner
+  def self.initialize_report_data
+    report_data = {}
+    report_data['productId'] = 'AWS SDK for Ruby v3'
+    begin
+      report_data['commitId'] = `git rev-parse HEAD`.strip
+    rescue StandardError
```

Comment:
This is a very vague rescue. We should instead check the directory and be smarter about whether we are in the repo root or not. Typically we don't use exceptions for control flow especially since 'standard error' can be just about anything.

Reply:
I think this snippet was pulled from the existing benchmark code: https://github.com/aws/aws-sdk-ruby-benchmark-tools/blob/main/benchmark.rb#L71-L75

Maybe worth using the `Git` gem ([repo](https://github.com/ruby-git/ruby-git)) instead of shelling out. Then, you could capture Git-specific errors

I agree with doing some checks on ensure we are in root repo.

Reply:
We could. I think shelling out is fine too since we do that everywhere else.